src/components/logger/request-log.ts

```diff
@@ -108,16 +108,9 @@ export class RequestLog {
     }ms`
     // generate logs from ping request or probe request
     probeMsg = this.request?.ping
-      ? `${this.iteration} id:${this.probe.id} PING:${
-          this.response?.alive ? 'alive' : 'dead'
-        } ${this.request?.url} ${
-          this.response?.alive ? this.response?.responseTime : '-'
-        }ms packetLoss:${
-          this.response?.alive ? this.response?.packetLoss : '-'
-        }%`
-      : `${this.iteration} id:${this.probe.id} ${
-          this.response?.status || '-'
-        } ${this.request?.method} ${this.request?.url} ${
+      ? `${this.iteration} id:${this.probe.id} ${this.response?.body}`
+      : `${this.iteration} id:${this.probe.id} ${this.response?.status || '-'} 
+      ${this.request?.method} ${this.request?.url} ${
```

Comment:
This code should be 

```javascript
if (this.request?.ping) {
   probeMsg = ...
}
```

because `probeMsg` already set with value in line 104.